class FailedConnectionError(Exception):
    """Raise for failed connection to miner"""


class AuthenticationError(Exception):
    """Raise for authentication failure to miner"""

